packages/fxa-settings/src/pages/Signin/en.ftl

```diff
@@ -0,0 +1,27 @@
+## Signin page
+
+# Strings within the <span> elements appear as a subheading.
+signin-password-needed-header = Enter your password <span>for your { -product-firefox-account }</span>
+
+# $serviceLogo - an image of the logo of the service which the user is authenticating for.
+# For languages structured like English, the phrase can read "to continue to"
+signin-subheader-with-logo = Continue to <span>{ $serviceLogo }</span>
+
+# $serviceName - the name of the service which the user authenticating for
+# For languages structured like English, the phrase can read "to continue to { $serviceName }"
+signin-subheader-without-logo-with-servicename = Continue to { $serviceName }
+signin-subheader-without-logo-default = Continue to account settings
+signin-button = Sign in
+signin-header = Sign in
+
+# This message is followed by a bulleted list
+signin-tos-list-intro = By proceeding, you agree to:
+signin-tos-list-pocket = { product-pocket }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
+signin-tos-list-firefox = { -brand-firefox }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal># <linkExternal> links to the Terms of Service and also to the Privacy Notice
```

Comment:
```suggestion
signin-tos-list-firefox = { -brand-firefox }’s <linkExternal>Terms of Service</linkExternal> and <linkExternal>Privacy Notice</linkExternal>
# <linkExternal> links to the Terms of Service and also to the Privacy Notice
```
Looks like a line break got removed. This is the only blocking issue (one more comment regarding ID names below).

Reply:
oh shoot I'll fix that 😮‍💨